llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -1082,21 +1082,29 @@ numGlobalVarSummaries(const ModuleSummaryIndex &Index,
   return NumGVS;
 }
 
-// Given ImportMap, return the number of global variable summaries and record
-// the number of defined function summaries as output parameter.
-static unsigned
-numGlobalVarSummaries(const ModuleSummaryIndex &Index,
-                      const FunctionImporter::FunctionsToImportTy &ImportMap,
-                      unsigned &DefinedFS) {
+struct ImportSummary {
```

Comment:
nit: I don't feel too strongly, but Summary is also used in the sense of GlobalVarSummary, FunctionSummary, etc. and so I wonder if using a different term like "Statistics" might be better.

Reply:
Agree, the word summary ends up being overloaded - I like "Statistics" (here and in the uses where the stats get dumped later on) better.

Reply:
I've renamed `ImportSummary` to `ImportStatistics`.  Also, I've renamed `summarizeImports` to `collectImportStatistics`.  Thanks!